src/main/java/com/example/kidz_school/service/impl/ConfirmationTokenServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.example.kidz_school.service.impl;
+
+import com.example.kidz_school.config.properties.ConfirmationTokenConfigProperties;
+import com.example.kidz_school.entity.ConfirmationToken;
+import com.example.kidz_school.entity.User;
+import com.example.kidz_school.repository.ConfirmationTokenRepository;
+import com.example.kidz_school.service.ConfirmationTokenService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class ConfirmationTokenServiceImpl implements ConfirmationTokenService {
+    private final ConfirmationTokenRepository confirmationTokenRepository;
+
+    //?????????????????????@Schedulllllllllllllllllllll
+    public void deleteExpiredTokens() {
+        LocalDateTime currentDateTime = LocalDateTime.now();
+        List<ConfirmationToken> expiredTokens = confirmationTokenRepository.findByExpiresAtBefore(currentDateTime);
+
+        confirmationTokenRepository.deleteAll(expiredTokens);
+    }
+
```

Comment:
Why are you performing two database operations? What's the purpose of fetching the data and then removing it? Why couldn't you just perform removeByExpiresAtBefore?